### PR TITLE
[Enhancement] Add bonus to views and some renamings

### PR DESCRIPTION
**Pull request by fde312**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) locally.
- [x] Client: I added multiple screenshots/screencasts of my UI changes
- [x] Client: I translated all the newly inserted strings into German and English

### Motivation and Context

As we added bonus points to exercises, we want to display them as well in different views.

### Description

- Added bonus points to some views
- Changed some translations (Exercise Type -> Type, Max Points -> Points)

### Steps for Testing

1. Log in to Artemis
2. Navigate to Course Administration and 
3. Go to Assessment/Instructor Dashboard of a course and then to Exercise Dashboard 
4. Check if for exercises with bonus points the bonus points are displayed as well also (Exercise Type -> Type, Max Points -> Points)
5. Navigate to Course Overview and go into a course
6. Enter a Exercise and check if for exercises with bonus points the bonus points are displayed as well
7. Go to Course Administration and then to Exams
8. Check Exercise Groups and check that it says Points and not Max. Points
9. Participate in Exam (or do a test run) check if for exercises (modeling, text and programming) with bonus points the bonus points are displayed as well

Check also if I missed some other spots where this can be changed/added (see also comments in discussion). Just let me know :) 
### Test Coverage

Nothing changed

### Screenshots

Student Exercise Details View:
Bonus:
image

No Bonus:
image

Assessment Dashboard:
image

Instructor Dashboard:
image

Exam:
image

image

### CODE REVIEW

**Comment by fde312** [comment]
Client tests are failing because of formatting. I just runned prettier and got this back:
image

So do not wonder why it is formatted so strangely 😅

**Review by MaisiKoleni — COMMENTED** [review]
Looks good and works well (for both German and English).

I just have some points that are unclear to me:
- Do we want to display bonus points in the "View" exercise as well? Currently, there are only the normal points.
- For exams (in this case a test run) i found "Max. Points" in the summary. Is this intended/ok?
  image

File: src/main/webapp/app/exam/participate/exercises/modeling/modeling-exam-submission.component.html [hunk 1]
```diff
@@ -1,7 +1,11 @@
 <jhi-resizeable-container class="col-12" *ngIf="exercise">
     <!--region Left Panel-->
     <span left-header *ngIf="exercise.title">{{ exercise.title }}</span>
-    <span left-header>&nbsp;[{{ exercise.maxScore }} {{ 'artemisApp.examParticipation.points' | translate }}]</span>
+    <span left-header
+        >&nbsp;[{{ exercise.maxScore }} {{ 'artemisApp.examParticipation.points' | translate }}
```
Comment (FrankeLukas) [inline src/main/webapp/app/exam/participate/exercises/modeling/modeling-exam-submission.component.html:5]: Can we move the closing tags onto the same line? (Here and in the files below.) Imho
```
<span
    >
```
looks very weird compared to just having `<span>` when there are no attributes.

**Review by FrankeLukas — COMMENTED** [review]
Imho
grafik
would look better as
grafik
Opinions?

**Comment by krusche** [comment]
> Imho
> grafik
> would look better as
> grafik
> Opinions?

I would like to avoid a third line and rather make the header a bit smaller. I think the current display is good

**Comment by krusche** [comment]
> * Do we want to display bonus points in the "View" exercise as well? Currently, there are only the normal points.
Yes, this would be good!
> * For exams (in this case a test run) i found "Max. Points" in the summary. Is this intended/ok?
Please rename to Points here as well.

Basically Max. Points should not be used any more in the whole application

**Review by FuchsDominik — CHANGES_REQUESTED** [review]
In my exam summary, the bonus points are not shown. (Exercise 1 should have bonus points)
image
Mini remark: You could switch the white space to the second string, so without the second string, the square brackets are even:
image
The rest looked very good to me!

**Comment by fde312** [comment]
> In my exam summary, the bonus points are not shown. (Exercise 1 should have bonus points)
> image
> Mini remark: You could switch the white space to the second string, so without the second string, the square brackets are even:
> image
> The rest looked very good to me!

@FuchsDominik I added this now:
image

The issue with the whitespace comes from the formatting. I tried out to use prettier-ignore to fix the issue with the styling mentioned by @FrankeLukas. 

@MaisiKoleni I added the bonus points also in the exercise view :

image

Would be nice if you could recheck :)

**Review by krusche — APPROVED** [review]

**Pull request merged**